bin/ka-clone

```diff
@@ -266,6 +280,25 @@ def install_global_git_hooks():
     _install_git_hook('commit-msg', 'commit-msg')
 
 
+def run_webapp_make_hooks():
+    _cli_log_step_success("Running make hooks because this repo is webapp...")
+    # Must run in top-level folder, even if this is called in subdirectory
+
+    p = subprocess.Popen(
+        ['make', 'hooks'],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        cwd=_top_level_dir()
+    )
+    out, err = p.communicate()
+
+    if p.returncode != 0:
+        _cli_log_step_indented_info("Failed to run make hooks")
+        _cli_log_step_indented_info("Error: {}".format(err.decode('utf-8')))
+    else:
+        _cli_log_step_indented_info("Ran make hooks successfully")
```

Comment:
I think the pythonic way to do this is:
```
try:
    subprocess.check_output(['make', 'hook'], stderr=subprocess.STDOUT, cwd=_top_level_dir())
    _cli_log_step_indented_info("Ran make hooks successfully")
except subprocess.CalledProcessError as e:
    _cli_log_step_indented_info("Failed to run make hooks")
    _cli_log_step_indented_info("Error: {}".format(e.output.decode('utf-8')))
```
Alternately, you could leave out the `stderr=STDOUT` bit, and have it just emit the error itself, rather than in the ` _cli_log_step_indented_info("Error: {}".format(e.output.decode('utf-8')))` line.